scripts/timezone.js

```diff
@@ -0,0 +1,74 @@
+const moment = require("moment-timezone");
+const utils = require("../utils");
+
+const TIMEZONES = {
+  at: "America/Puerto_Rico",
+  adt: "America/Puerto_Rico",
+  ast: "America/Puerto_Rico",
+  ct: "America/Chicago",
+  cdt: "America/Chicago",
+  cst: "America/Chicago",
+  et: "America/New_York",
+  edt: "America/New_York",
+  est: "America/New_York",
+  mt: "America/Denver",
+  mdt: "America/Denver",
+  mst: "America/Denver",
+  pt: "America/Los_Angeles",
+  pdt: "America/Los_Angeles",
```

Comment:
what about Alaska?

Reply:
For Mark Meyer 🐻

Reply:
Yessss, good catch! I did check with Edwin to see if there are common shorthands in Puerto Rico, but he said there aren't. There are for Alaska, though! AKT, AKDT, AKST! Added those.